src/audio/src/coef/src_lite_ipc4_int32_define.h

```diff
@@ -0,0 +1,23 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2023 Intel Corporation. All rights reserved.
+ *
+ * Author: Fabiola Jasinska <fabiola.jasinska@intel.com>
+ */
+
+#ifndef __SOF_AUDIO_COEFFICIENTS_SRC_SRC_IPC4_INT32_DEFINE_H__
+#define __SOF_AUDIO_COEFFICIENTS_SRC_SRC_IPC4_INT32_DEFINE_H__
+
+/* SRC constants */
+#define MAX_FIR_DELAY_SIZE 730
+#define MAX_OUT_DELAY_SIZE 900
+#define MAX_BLK_IN 80
+#define MAX_BLK_OUT 40
+#define NUM_IN_FS 16
+#define NUM_OUT_FS 10
+#define STAGE1_TIMES_MAX 32
+#define STAGE2_TIMES_MAX 32
+#define STAGE_BUF_SIZE 672
+#define NUM_ALL_COEFFICIENTS 69224
```

Comment:
I would prefix these with SRC_ as some names are quite common, but I think these are added before this commit so we should revisit this @singalsu ?

Reply:
Yes you are right, I can do it after this PR. These files are automatically generated so better to do the change to Matlab code also.

Reply:
ok, thaks @singalsu let add the prefix after this PR is merged.